### PR TITLE
Note that PackageIconUrl is deprecated

### LINKED ISSUE #16514: PackageIconUrl
What is there no deprecation mentioning about PackageIconUrl?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: cb4428fc-215c-78e1-5ff0-fe73f7a08260
* Version Independent ID: 6741b1de-38a4-fbb1-d79d-acc47d549a2d
* Content: [Additions to the csproj format for .NET Core - .NET Core CLI](https://docs.microsoft.com/en-us/dotnet/core/tools/csproj#feedback)
* Content Source: [docs/core/tools/csproj.md](https://github.com/dotnet/docs/blob/master/docs/core/tools/csproj.md)
* Product: **dotnet-core**
* Technology: **dotnet-cli**
* GitHub Login: @tdykstra
* Microsoft Alias: **tdykstra**

### PR DESCRIPTION
**Pull request by tdykstra**
Fixes #16514

### CODE REVIEW

**Review by CamSoper — APPROVED** [review]

File: docs/core/tools/csproj.md [hunk 1]
```diff
@@ -342,9 +342,9 @@ You will need to ensure the license file is packed by adding it explicitly to th
 
 A URL to the license that is applicable to the package. (_deprecated since Visual Studio 15.9.4, .NET SDK 2.1.502 and 2.2.101_)
 
-### PackageIconUrl
+### PackageIcon
 
-A URL for a 64x64 image with transparent background to use as the icon for the package in UI display.
+A URL for a 64x64 image with transparent background to use as the icon for the package in UI display. [PackageIconUrl is deprecated](/nuget/reference/msbuild-targets#packageiconurl) in favor of PackageIcon.
```
Comment (Youssef1313) [inline docs/core/tools/csproj.md:347]:
```suggestion
A path to an image in the package to use as a package icon. Read more about [`icon` metadata](/nuget/reference/nuspec#icon). [PackageIconUrl is deprecated](/nuget/reference/msbuild-targets#packageiconurl) in favor of PackageIcon.
```

File: docs/core/tools/project-json-to-csproj.md [hunk 2]
```diff
@@ -472,7 +472,7 @@ See also [Files](#files).
   <!-- summary is not migrated from project.json, but you can use the <Description> property for that if needed. -->
   <PackageTags>machine learning;framework</PackageTags>
   <PackageReleaseNotes>Version 0.9.12-beta</PackageReleaseNotes>
-  <PackageIconUrl>http://numl.net/images/ico.png</PackageIconUrl>
+  <PackageIcon>http://numl.net/images/ico.png</PackageIconUrl>
```
Comment (Youssef1313) [inline docs/core/tools/project-json-to-csproj.md:475]:
```suggestion
  <PackageIcon>http://numl.net/images/ico.png</PackageIcon>
```

Comment (Youssef1313) [inline docs/core/tools/project-json-to-csproj.md:475]: I think this shouldn't use a URL at all, based on @JamesNK suggestion in https://github.com/dotnet/docs/pull/19628

Comment (Youssef1313) [reply docs/core/tools/project-json-to-csproj.md:475]: #19746 might be related here too.

Comment (tdykstra) [reply docs/core/tools/project-json-to-csproj.md:475]: What would the typical path-to-file be here then for an example?  Should we put just the file name?

Comment (Youssef1313) [reply docs/core/tools/project-json-to-csproj.md]: The examples I've seen were only file names.
But I dont know if PackageIconFullPath property is required or not.

**Comment by tdykstra** [comment]
Thanks for the corrections, @Youssef1313 !

**Pull request merged**